sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/VirtualMachineScaleSetVMsOperations.cs

```diff
@@ -3367,7 +3367,7 @@ public async Task<AzureOperationResponse<RunCommandResult>> BeginRunCommandWithH
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "this.Client.SubscriptionId");
             }
-            string apiVersion = "2021-03-01";
+            string apiVersion = "2020-12-01";
```

Comment:
Using an older version of Swagger to generate this. Incorrect.

Reply:
fixed